test/3x/torch/quantization/weight_only/test_transfomers.py

```diff
@@ -81,8 +85,10 @@ def test_quantization_for_llm(self):
         woq_model = AutoModelForCausalLM.from_pretrained(model_name_or_path, quantization_config=woq_config)
         woq_model.eval()
         output = woq_model(dummy_input)
-        assert isclose(float(output[0][0][0][0]), 0.17234990000724792, rel_tol=1e-04)
-
+        if ipex_version < Version("2.5.0"):
+            assert isclose(float(output[0][0][0][0]), 0.17234990000724792, rel_tol=1e-04)
+        else:
+            assert isclose(float(output[0][0][0][0]), 0.17049233615398407, rel_tol=1e-04)
```

Comment:
Do you have a explanation why the output changes due to torch version?

Reply:
Since the output of torch.cholesky() has changed in different Torch versions. updated into code https://github.com/intel/neural-compressor/pull/2066/commits/8bf432dd071944a494d836af9c64401c56069d66